downgrade

```diff
@@ -484,5 +468,22 @@ if ((!LIB)); then
 
   export TEXTDOMAIN=downgrade TEXTDOMAINDIR
 
+  # check for help CLI option
+  for arg; do
+    if [[ "$arg" =~ ^-h$|^--help$ ]]; then
+      usage
+      exit 1
```

Comment:
Previous behavior, but I don't think `-h|--help` should exit unsuccessfully. The program did succeed in printing usage as asked.

Printing usage in response to an invalid argument should be non-zero, which is I think why this got this way by mistake.

Reply:
Ah yes, changed this.